### PR TITLE
Adding the area

### PR DESCRIPTION
**Pull request by ISHHMadushanka**

### CODE REVIEW

File: filters.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
```
Comment (ISHHMadushanka) [inline filters.html:1]: fixed area

File: filters.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                   'ityup',
+                   'jhtyii',
+
+                ];
+
+       	  });
+
+       </script>      	
+</body>
+</html>
```
Comment (ISHHMadushanka) [inline filters.html:44]: find area of any those thing

**Comment by AshuKulu** [comment]
Thank You
Star this Repo
Keep Contributing

**Comment by ISHHMadushanka** [comment]
sure ,i will do. thank you!!!

**Pull request merged**